dev/a11y_assessments/lib/use_cases/snack_bar.dart

```diff
@@ -29,6 +30,22 @@ class MainWidget extends StatefulWidget {
 class MainWidgetState extends State<MainWidget> {
   String pageTitle = getUseCaseName(SnackBarUseCase());
 
+  void showAccessibleSnackBar(String message, {VoidCallback? onAction}) {
+    ScaffoldMessenger.of(context).showSnackBar(
+      SnackBar(
+        content: Text(message),
+        action: onAction != null
+            ? SnackBarAction(label: 'Action', onPressed: onAction)
+            : null,
+      ),
+    );
+
+    Future<void>.delayed(const Duration(milliseconds: 100), () {
```

Comment:
we should only do this for MediaQueryData.supportsAnnounce is true, you can get this from MediaQuery.of(context).